live-examples/html-examples/text-content/css/blockquote.css

```diff
@@ -0,0 +1,12 @@
+blockquote {
+  background: #eee;
+  border-radius: 5px;
+}
+blockquote p {
+	padding: 15px;
+}
+
+blockquote p::before,
+blockquote p::after {
+	content: '\201C';
```

Comment:
This character is [directional](https://www.fileformat.info/info/unicode/char/201c/index.htm), so I think you'll want:

```
blockquote p::before {
	content: '\201C';
}

blockquote p::after {
  content: '\201D';
}
```

Reply:
...only with less random indentation, I guess :-)